### PR TITLE
Pythonic reserved names

### PR DESCRIPTION
**Pull request by graeme-winter**
Avoid python reserved names as variables in source code - have avoided wx which seems to overload the use of id by default but otherwise we have a few hand-tweaked names which should be idiomatically sensible.

### CODE REVIEW

**Comment by graeme-winter** [comment]
Obviously squash and merge would be tidier

File: util/image.py [hunk 1]
```diff
@@ -62,7 +62,7 @@ def get_data(self):
 
 if __name__ == "__main__":
     import sys
-    import numpy
+    import numpy  # noqa: F401
```
Comment (ndevenish) [inline util/image.py]: Is this necessary at all?

File: util/intensity_explorer.py [hunk 2]
```diff
@@ -392,10 +392,10 @@ def data_from_unmerged_mtz(filename):
     rtable["id"] = flex.int(rtable.size(), 0)
 
     # Now generate a corresponding experiment list.
-    id = flex.vec3_double([(1, 0, 0), (0, 1, 0), (0, 0, 1)])
+    _id = flex.vec3_double([(1, 0, 0), (0, 1, 0), (0, 0, 1)])
```
Comment (ndevenish) [inline util/intensity_explorer.py]: This appears to be using miller index as an id - so `indices` might be better? or even just `ids`...

**Comment by ndevenish** [comment]
I rewrote some of the names to be what I thought were better choices. e.g. things like instead of renaming `file`→`f`, rename to `filename` because that's what it is. 

I'm fine with everything now.

**Pull request merged**